client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/contact-editor/hooks.tsx

```diff
@@ -86,21 +86,27 @@ export function useContactFormInputHelpText( type: AllowedMonitorContactTypes )
 	}, [ translate, type ] );
 }
 
+const EMPTY_ARRAY: SmsCountry[] = [];
+
 export function useGetSupportedSMSCountries() {
-	// Fetch only once
-	const { data: geoData } = useGeoLocationQuery( {
-		refetchOnWindowFocus: false,
-		staleTime: Infinity,
-	} );
```

Comment:
I removed this extra configuration because it will be soon obsoleted by #102759 which applies similar options globally, to the `useGeoLocationQuery` hook itself.